mantidproject/mantid
# Merge pull request #27747 from mantidproject/27734_move_grouping_from_params

**Description of work.**
Moves the tube-by-tube grouping pattern from instrument parameters to a grouping file.
This should have no effect on the results, but apparently one bin fluctuates slightly after this change, causing the tests to fail with stringent tolerances. The tolerances are relaxed, while in commissioning phase.



**To test:**
LoadEmptyInstrument of D33, go to pick tab of instrument view, and mouse hover here and there, it should not be freezing, nor lagging.


**Reviewer**
Merge also [PR #161](https://github.com/mantidproject/mantidgeometry/pull/161) in `mantidgeometry`. 


Fixes #27734 . 



*This does not require release notes, because it is an internal move of the parameter.*


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
